test/ApplicationFactory.ts

```diff
@@ -45,10 +48,16 @@ export const createApp = (data?: Partial<ApplicationData>,
 
         const sessionId = session?.data[SessionKey.Id];
         const signature = session?.data[SessionKey.ClientSig];
-
         const cookie = session ? Cookie.createFrom(sessionId! + signature) : null;
-
         const sessionStore = Substitute.for<SessionStore>();
+        const sessionConfig: SessionStoreConfig  = SessionStoreConfig.createFromEnvironmentVariables();
+        const encryptionService = new JwtEncryptionService();
+        const companyAuthConfig: CompanyAuthConfig = {
+            accountUrl: 'localhost',
+            accountRequestKey: 'pXf+qkU6P6SAoY2lKW0FtKMS4PylaNA3pY2sUQxNFDk=',
```

Comment:
Lets's use different value. @akaJSaunders put a test where key is created by repeating 'a' number of times.